src/packages/@ncigdc/modern_components/SCRNASeq/SCRNASeq.js

```diff
@@ -86,19 +129,35 @@ const SCRNASeq = ({
 
         <Row>
           {/* <ToolbarButton
-          //   faClass="fa-angle-double-down"
-          //   label="Get TSV"
-          //   name="downloadAnalysis"
-          //   onToolbarClick={getWholeTsv}
-          //  />
+            //   faClass="fa-angle-double-down"
+            //   label="Get TSV"
+            //   name="downloadAnalysis"
+            //   onToolbarClick={getWholeTsv}
+            //  />
           */}
-
           <DownloadButton
             onAnalysisClick={handleAnalysisClick}
             {...buttonList.downloadAnalysis}
             />
         </Row>
       </Row>
+      <Table
+        body={(
+          <tbody>
+            {analysisTable.map((row, i) => (
+              <Tr index={i} key={row.name}>
+                <Td><strong>{row.name}</strong></Td>
+                <Td>
+                  {row.link
+                    ? <a href={row.link}>{row.text}</a>
+                    : row.text}
+                </Td>
+              </Tr>
+            ))}
+          </tbody>
+        )}
+        style={{ marginBottom: 20, maxWidth: 500 }}
+        />
```

Comment:
```suggestion
      {analysisTable.length && (
        <Table
          body={(
            <tbody>
              {analysisTable.map((row, i) => (
                <Tr index={i} key={row.name}>
                  <Td><strong>{row.name}</strong></Td>
                  <Td>
                    {row.link
                      ? <a href={row.link}>{row.text}</a>
                      : row.text}
                  </Td>
                </Tr>
              ))}
            </tbody>
          )}
          style={{ marginBottom: 20, maxWidth: 500 }}
          />
        )}
```

Reply:
This change, in case no array is being passed for whatever reason.
It may be useful to make it a ternary, rather than an &&, so we can display something instead in that scenario.

Reply:
Good idea! I'll add some error handling.

Reply:
after discussing on slack, this info is guaranteed to be there (in the current implementation)